CMakeLists.txt

```diff
@@ -1,171 +1,104 @@
-# print initial information about the project
 message("Running CMake on libdnf...")
-project(libdnf C CXX)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Let's make this minimum CMake 3.5. We could go up to CMake 3.10 (which is what openSUSE Leap 15.0 ships and is the oldest of the currently supported distros with DNF). But CMake 3.0 is too restrictive.

Reply:
Makes sense. Setting cmake 3.5.